src/libraries/Microsoft.PowerFx.Core/Public/DefinitionsCheckResult.cs

```diff
@@ -49,9 +51,10 @@ public class DefinitionsCheckResult : IOperationStatus
         // All errors accumulated. 
         private readonly List<ExpressionError> _errors = new List<ExpressionError>();
 
-        public DefinitionsCheckResult()
+        public DefinitionsCheckResult(Features features = null)
```

Comment:
>DefinitionsCheckResult [](http://example.com/codeflow?start=15&length=22)

nit - if you were trying to do this in a non-breaking way (see warnings in PR) 

you could add a new ctor, and have existing one chain to it:

DefinitionsCheckResult() : this(Features.PowerFxV1)

DefinitionsCheckResult(Features features) { }